src/lib/stitching/convection/schema.ts

```diff
@@ -0,0 +1,35 @@
+import { createConvectionLink } from "./link"
+import {
+  makeRemoteExecutableSchema,
+  transformSchema,
+  RenameTypes,
+} from "graphql-tools"
+import { readFileSync } from "fs"
+
+export const executableConvectionSchema = async () => {
+  const convectionLink = createConvectionLink()
+  const convectionTypeDefs = readFileSync("src/data/convection.graphql", "utf8")
+
+  // Setup the default Schema
+  const schema = await makeRemoteExecutableSchema({
+    schema: convectionTypeDefs,
+    link: convectionLink,
+  })
+
+  // Remap the names of certain types from Convection to fit in the larger
+  // metaphysics ecosystem.
+  const remap = {
+    Submission: "ConsignmentSubmission",
+    Category: "SubmissionCategoryAggregation",
```

Comment:
Curious why the `Aggregation` naming? But 👍 love this!

Reply:
That was the original name in metaphysics, can't remember why it's called that now, but likely was replicating something on convection